src/GafferSceneUI/SceneView.cpp

```diff
@@ -781,14 +783,29 @@ class SceneView::LookThrough : public boost::signals::trackable
 			lookThrough->addChild( new StringPlug( "camera", Plug::In, "", Plug::Default & ~Plug::AcceptsInputs ) );
 			view->addChild( lookThrough );
 
-			// Set up our nodes. We use a standard options node to disable camera motion blur
+			// Set up our nodes.
+			// We use a LightToCamera node filtered to all lights to create camera standins so that we can
+			// look through lights
+			SetFilterPtr lightFilter = new SetFilter;
+			lightFilter->setPlug()->setValue( "__lights" );
+			
+			LightToCameraPtr lightConverter = new LightToCamera;
+			lightConverter->inPlug()->setInput( view->inPlug<ScenePlug>() );
+			lightConverter->filterPlug()->setInput( lightFilter->outPlug() );
+
+			m_internalNodes.push_back( lightFilter );
+			m_internalNodes.push_back( lightConverter );
+
+			
```

Comment:
Could you limit blank lines to one at a time? Splitting things for readability is great, but I'd like to balance it with pushing things off screen vertically.